### PR TITLE
#38 create wallet screen

### PR DESCRIPTION
**Pull request by omarodev**

### CODE REVIEW

File: pages/welcome.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+.main-section {
+  padding: 15px 10px 35px;
+  background: rgba(41, 41, 41, 0.6);
+  margin: 0 25px;
+  border-radius: 3px;
+  align-items: center;
+}
+.main-section .welcome-section {
+  text-transform: uppercase;
+  font-size: 2.8rem;
```
Comment (shawnlauzon) [inline pages/welcome.vue]: There should be no need for font sizes here; use Bulma sizes

**Comment by shawnlauzon** [comment]
BTW, this is supposed to be first pass, and so I would rather it be less perfect but with much less CSS. We can always improve it later.

**Comment by shawnlauzon** [comment]
I still see lots of pixels in the CSS. And disabling resizing the desktop is not acceptable.

**Comment by shawnlauzon** [comment]
Mostly good:

image'

Biggest problem is that according to the design, opacity of that box should be 42%, and I believe your opacity is less.

image

You can [refer to the specifications here](https://xd.adobe.com/spec/235e0e32-283a-45dd-4557-d76b26d250dd-556f/)

**Comment by omarodev** [comment]
Thank you for your review. 👍  

I will update the overlay. Btw, could you send me design file today?

**Comment by shawnlauzon** [comment]
[Design file is here](https://xd.adobe.com/spec/235e0e32-283a-45dd-4557-d76b26d250dd-556f/)

**Pull request merged**